main/templates/main/partials/process_table.html

```diff
@@ -0,0 +1,4 @@
+{% load render_table from django_tables2 %}
+<div hx-post="{% url 'main:process_table' %}"
+     hx-trigger="load delay:1s, click from:input"
+     hx-swap="outerHTML">{% render_table table %}</div>
```

Comment:
So `render_table` is substituted immediately on page load with a bootstrap div. Then here the new div's trigger fires 1 sec after load, substituting into itself, re-rendering the table and restarting the trigger timer. `process_table` is in its own page allowing HTMX to just call it, rather than `index`. Is `hx-post` used to send checkbox state, rather than `hx-get`?